src/contents/shared/ProfileEdit.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Component } from 'react';
+import fire from '../../config/Fire';
+import { ProfileEditView } from './ProfileEditView';
+import firebase from 'firebase';
+import { storage } from '../../config/Fire';
+
+export class ProfileEdit extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+		  image: "https://example.com/jane-q-user/profile.jpg",
+		  url: this.props.user.photoURL,
+		  progress: 0,
+		  name: this.props.user.displayName,
+		  email: this.props.user.email,
+		  type: this.props.user.type,
+		  location: this.props.user.location,
+		  photoURL: this.props.user.url,
+		}
+		this.handleChangeUpload = this
+      .handleChangeUpload
+      .bind(this);
+       
+	  }
+
+
+	database = fire.database().ref();
+
+	/**
+		* Updates state to the current value of a certain target.
+		* @param e
+		*/
+		handleChange = e => {
+			this.setState({ [e.target.name]: e.target.value });
+		};
+
+	/**
+		* Registers the user as photographer or company.
+		*
+		* @param e
+		*/
+
+	updateUser = (e) => {
+		e.preventDefault();
+		let { name, email, type, location, url } = this.state;
+		const { user } = this.props;
+
+		if (name !== "" && email !== "" && location !== "" && url !== "") {
+
+		var newUser = firebase.auth().currentUser;
```

Comment:
we already save the current user in the props (line 47), why are you creating a new one? Also please always use let or const instead of var

Reply:
Done!